### DISCUSSION TITLE
Bottom menu doesn't show all the options as shown in the wiki, when in portait mode.

### DISCUSSION THREAD

**Discussion by razzat008** [koreader/koreader#10214] [Q&A]
koreader version=2023.01(95512)

![photo_2023-03-17_08-55-56](https://user-images.githubusercontent.com/93041325/225803243-c7b0883b-76d3-45d6-86b7-be15bef9ce0a.jpg)

**Comment by Frenzie** [comment]
Some Android that misreports things?

**Reply by razzat008** [reply]
Is this a bug or am I missing some settings?

**Reply by poire-z** [reply]
You're missing telling us what's your device :) (and Android version if applicable).
If Android, see https://github.com/koreader/koreader/issues/9446#issuecomment-1297542495

**Reply by mergen3107** [reply]
Your screen is cropped probably
